jobs/build/release/Jenkinsfile

```diff
@@ -82,6 +91,19 @@ node {
                         description: 'DO NOT USE without team lead approval. Allows the pipeline to overwrite an existing payload in quay.',
                         defaultValue: false,
                     ),
+                    choice(
+                        name: 'ENABLE_AUTOMATION',
+                        description: [
+                                'Unfreeze automation to enable building and sweeping into the new advisories',
+                                '<b>Default</b>: Enable for Standard and RC releases, if arch is x86_64',
+                            ].join('\n'),
+                        choices: [
+                            'Default',
+                            'Yes',
+                            'No',
+                        ].join('\n'),
+                        defaultValue: true
```

Comment:
`choice` doesn't really have a `defaultValue` does it? anyway it wouldn't be boolean here